### PR TITLE
min borrow should be bigger

### PR DESCRIPTION
**Pull request by smartdevcom**
Make min borrow value a little bigger than real one

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "dev:node": "yarn workspace @midas-capital/sdk hardhat node",
     "dev:fork:chapel:node": "FORK_URL_BSC=${MY_CHAPEL_FORK_URL} FORK_BLOCK_NUMBER= FORK_CHAIN_ID=97 yarn workspace @midas-capital/sdk hardhat node --no-deploy --tags fork",
     "dev:fork:bsc:node": "FORK_URL_BSC=${MY_BSC_FORK_URL} FORK_BLOCK_NUMBER= FORK_CHAIN_ID=56 yarn workspace @midas-capital/sdk hardhat node --no-deploy --tags fork",
+    "dev:fork:polygon:node": "FORK_URL_POLYGON=${MY_POLYGON_FORK_URL} FORK_BLOCK_NUMBER= FORK_CHAIN_ID=137 yarn workspace @midas-capital/sdk hardhat node --no-deploy --tags fork",
```
Comment (peetzweg) [inline package.json:49]:
```suggestion
    "dev:fork:polygon:node": "FORK_URL_BSC=${MY_POLYGON_FORK_URL} FORK_BLOCK_NUMBER= FORK_CHAIN_ID=137 yarn workspace @midas-capital/sdk hardhat node --no-deploy --tags fork",
```

The env variable used throughout hh config and docker is called `FORK_URL_BSC` so it's branded with BSC, which is stems from the first time we introduced fork. 
When we set `FORK_URL_POLYGON` it's not used anywhere. So we need to use  `FORK_URL_BSC` instead.

File: packages/ui/utils/formatNumber.ts [hunk 2]
```diff
@@ -20,3 +20,7 @@ export const bigMul = (value1: string, value2: string) => {
   const val = multiply(bignumber(value1), bignumber(value2));
   return format(val, { notation: 'fixed' });
 };
+
+export const toCeil = (value: number, len: number) => {
```
Comment (peetzweg) [inline packages/ui/utils/formatNumber.ts:24]: Nice. 👍 
Sad thought that we don't have a test suite to add some unit test for this function.

File: package.json (hunk 1 above)
Comment (smartdevcom) [reply package.json]: fixed

File: packages/ui/utils/formatNumber.ts (hunk 2 above)
Comment (smartdevcom) [reply packages/ui/utils/formatNumber.ts:24]: I will add it later on

**Pull request merged**